### PR TITLE
add kashgari to readme

### PR DESCRIPTION
**Pull request by BrikerMan**
Kashgari is a Simple and powerful NLP framework, build your state-of-art model in 5 minutes for named entity recognition (NER), part-of-speech tagging (PoS) and text classification tasks. Includes BERT and word2vec embedding.

### CODE REVIEW

**Comment by NirantK** [comment]
Hey @BrikerMan -- this is quite interesting!

I don't see why this is simpler than AllenNLP or spaCy? Plus the SoTA is BiDAF++ and BiLSTM+CRF is quite old.

The Keras implementation is nice. Better than TF for lot of beginners at the cost of "powerful" perhaps - given any model implementation will be limited by Keras functionality? 

The lack of docs covering custom datasets leaves me confused. 
Overall, I am not yet confident if we'd like to recommend this broadly as "simple and powerful".  

This is not a broad criticism and hand-waving, but me trying to understand why this description is valid. Pardon my ignorance.

**Comment by BrikerMan** [comment]
I think it is simply because it is very easy to customize your own keras model like [this tutorial](https://github.com/BrikerMan/Kashgari/blob/master/docs/Tutorial_2_Classification.md#customize-your-own-model).
Powerful because you could use BERT with single line to improve your models performance.

**Comment by NirantK** [comment]
Hmm, still-not-SoTA? 

Using BERT in one line is something which AllenNLP etc have been doing a long while now. But I agree that is powerful. 

And as I mentioned, your own models are limited by Keras' functionality now. Simple, yes. Powerful, no?

**Comment by BrikerMan** [comment]
Yea. You are right. It is limit to keras models for now. The main focus of this project is it is very easy to use, learn and customize. I think you could remove the powerful from the description.

File: README.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ Bayesian, statistics and Linguistics approaches for Natural Language Processing
   - [Multilingual Latent Dirichlet Allocation (LDA)](https://github.com/ArtificiAI/Multilingual-Latent-Dirichlet-Allocation-LDA) - A multilingual and extensible document clustering pipeline
   - [NLP Architect](https://github.com/NervanaSystems/nlp-architect) - A library for exploring the state-of-the-art deep learning topologies and techniques for NLP and NLU
   - [Flair](https://github.com/zalandoresearch/flair) - A very simple framework for state-of-the-art multilingual NLP built on PyTorch. Includes BERT, ELMo and Flair embeddings.
+  - [Kashgari](https://github.com/BrikerMan/Kashgari) - Simple and powerful NLP framework, build your state-of-art model in 5 minutes for named entity recognition (NER), part-of-speech tagging (PoS) and text classification tasks. Includes BERT and word2vec embedding.
```
Comment (NirantK) [inline README.md]:
```suggestion
  - [Kashgari](https://github.com/BrikerMan/Kashgari) - Simple, Keras-powered multilingual NLP framework, allows you to build your models in 5 minutes for named entity recognition (NER), part-of-speech tagging (PoS) and text classification tasks. Includes BERT and word2vec embedding.
```

**Comment by BrikerMan** [comment]
I will add new model structures later to make it powerful. But for now, it is just simple.

**Comment by NirantK** [comment]
I've made a suggestion in the Pull Request itself. Does that work for you? If yes - please approve?

**Comment by BrikerMan** [comment]
It works for me. It is more accurate than my description, thanks.

**Comment by NirantK** [comment]
Thank you @BrikerMan for taking the time to add your work here! 

I'd love to contrib as well. Consider adding a How to Contribute Section to your README? So that I'd know where to begin?

**Comment by BrikerMan** [comment]
Yes~ I will add this part tomorrow, thanks a lot~

**Comment by BrikerMan** [comment]
@NirantK I have finished the contribute section, please check out the [CONTRIBUTING.md](https://github.com/BrikerMan/Kashgari/blob/master/CONTRIBUTING.md)

**Pull request merged**